### DISCUSSION TITLE
Use the embedded dashboard in connected-app-credentials mode

### DISCUSSION THREAD

**Discussion by tlp319** [mulesoft-catalyst/metrics-toolkit#175] [Q&A]
- In my scenario I must use the authentication mode connected-app-credentials because an SSO/MFA policy in enforced across the organization.
- I am able to make POST requests to the app to return the data that I need in CSV using the credentials of my connected app and our orgID. 
- Now my organization want to be able to access the information using the embedded dashboard.

I have enabled the dashboard feature, and I have put the appropriate credentials for the connected app in the app-dev.yaml file. After I launch the app, I can go to the base URL of the app in my web browser and pull-up the sign in page for the embedded dashboard. But when I try to sign-in using my Anypoint creds and orgID, the log-in fails. Is it possible to use the embedded dashboard while using the connected-app-credentials authentication mode?

**Comment by richardmckinley** [comment]
Hi @tlp319, 
The embedded dashboard does not currently support connect app authentication. It supports platform user only. 

The embedded dashboard is intended to be a view "quick start" view of what the toolkit offers, rather than a long term solution. 

As with the rest of the toolkit, the embedded dashboard source code is available for you to extend if desired. 

Thanks, 
Richard
